files/etc/init.d/pbr

```diff
@@ -417,45 +421,37 @@ get_text() {
 	esac
 	echo "$r"
 }
-
-process_url() {
-	local url="$1"
-	local dl_command dl_https_supported dl_temp_file
-# TODO: check for FILE schema and missing curl
-	if is_present 'curl'; then
-		dl_command="curl --silent --insecure"
-		dl_flag="-o"
-	elif is_present '/usr/libexec/wget-ssl'; then
-		dl_command="/usr/libexec/wget-ssl --no-check-certificate -q"
-		dl_flag="-O"
-	elif is_present wget && wget --version 2>/dev/null | grep -q "+https"; then
-		dl_command="wget --no-check-certificate -q"
-		dl_flag="-O"
-	else
-		dl_command="uclient-fetch --no-check-certificate -q"
-		dl_flag="-O"
-	fi
-	if curl --version 2>/dev/null | grep -q "Protocols: .*https.*" \
-		|| wget --version 2>/dev/null | grep -q "+ssl"; then
-		dl_https_supported=1
-	else
-		unset dl_https_supported
-	fi
-	while [ -z "$dl_temp_file" ] || [ -e "$dl_temp_file" ]; do
-		dl_temp_file="$(mktemp -u -q -t "${packageName}_tmp.XXXXXXXX")"
-	done
-	if is_url_file "$url" && ! is_present 'curl'; then
-		state add 'errorSummary' 'errorFileSchemaRequiresCurl' "$url"
-	elif is_url_https "$url" && [ -z "$dl_https_supported" ]; then
-		state add 'errorSummary' 'errorDownloadUrlNoHttps' "$url"
-	elif $dl_command "$url" "$dl_flag" "$dl_temp_file" 2>/dev/null; then
-		sed 'N;s/\n/ /;s/\s\+/ /g;' "$dl_temp_file"
-	else
-		state add 'errorSummary' 'errorDownloadUrl' "$url"
-	fi
-	rm -f "$dl_temp_file"
+set_dl(){
+	[ -x "$wget" ] && {
+		dl_schemes='ftp http'
+		dl_command='$wget -q --no-check-certificate'
```

Comment:
Please use double quotes for dl_command expressions.

Reply:
Yes, otherwise it does not work :rofl: